### ISSUE TITLE
update http://code.google.com/p/reflections/ to tell that 0.9.6 are in mvn repo

### ISSUE THREAD

**Issue by GoogleCodeExporter** [NickstaDB/reflections#95]
```
subj says it all :)
```

Original issue reported on code.google.com by `nino.mar...@gmail.com` on 2 Jan 2012 at 10:03

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
released 0.9.7-RC1. comments?
```

Original comment by `ronm...@gmail.com` on 2 May 2012 at 3:52

**Comment by GoogleCodeExporter** [comment]
```
fixed 0.9.7
```

Original comment by `ronm...@gmail.com` on 22 May 2012 at 7:47
- Changed state: **Fixed**
